packages/terra-popup/tests/nightwatch/components/ArrowPopupClosed.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Popup from '../../../lib/Popup';
+
+class ArrowPopupClosed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  componentDidMount() {
+    this.forceUpdate();
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    const buttonStyle = {
```

Comment:
Nit:

If you use it just once, don't use a variable. Just pass it straight in.